sormas-backend/src/main/java/de/symeda/sormas/backend/importexport/ImportFacadeEjb.java

```diff
@@ -233,7 +233,10 @@ public void generateEventImportTemplateFile() throws IOException {
 
 		char separator = configFacade.getCsvSeparator();
 
-		List<String> columnsToRemove = Arrays.asList(EventDto.SORMAS_TO_SORMAS_ORIGIN_INFO, EventDto.OWNERSHIP_HANDED_OVER);
+		ArrayList<String> columnsToRemove = new ArrayList<>(Arrays.asList(EventDto.SORMAS_TO_SORMAS_ORIGIN_INFO, EventDto.OWNERSHIP_HANDED_OVER));
```

Comment:
if no particular reason for doing this, please remove new ArrayList<> and add List to columnsToRemove field declaration

Reply:
`Arrays.asList` returns a fixed size list, so I cannot add another column after list creation. On the other hand side, I wanted to keep the handy syntax of `asList`, that is why I use the result to create an `ArrayList`.
Any better idea?